include/glow/Support/Support.h

```diff
@@ -23,6 +23,21 @@
 #include <iostream>
 #include <sstream>
 
+#ifdef _MSC_VER
```

Comment:
I prefer to work around the dynamic use and not declare GLOW_VLA which is not very readable.